pkg/full/opam

```diff
@@ -1,6 +1,6 @@
 opam-version: "1.2"
 name: "oml"
-version: "0.0.5"
+version: "0.0.6"
 available: [ ocaml-version >= "4.01" ]
```

Comment:
I think now it's 4.03.0 only?

```
# File "myocamlbuild.ml", line 170, characters 38-60:
# Error: Unbound value String.lowercase_ascii
# Command exited with code 2.
```

(from `opam pin add oml -k git .` on this branch)